test.js

```diff
@@ -1,35 +1,63 @@
-const fs = require('fs')
+const fs = require('fs-extra')
 const path = require('path')
 const walk = require('walk-sync').entries
+const difference = require('lodash.difference')
 const { supportedVersions } = require('./package.json')
+
 const contentDir = path.join(__dirname, 'content')
+const versions = fs.readdirSync(contentDir)
+const originalSourceLocale = 'en-US'
 
-test('includes source English content for all major versions', () => {
-  const fetchedMajors = fs.readdirSync(contentDir).sort()
-  const targetMajors = supportedVersions.sort()
-  expect(targetMajors).toEqual(fetchedMajors)
+describe('original content', () => {
+  test('exist for all supported versions', () => {
+    expect(supportedVersions.sort()).toEqual(versions.sort())
+    versions.forEach((major) => {
+      const languages = fs.readdirSync(path.join(contentDir, major))
+      expect(languages.includes(originalSourceLocale)).toBe(true)
+    })
+  })
 
-  fetchedMajors.forEach(major => {
-    const languages = fs.readdirSync(path.join(contentDir, major))
-    expect(languages.includes('en-US')).toBe(true)
+  test('includes only markdown files', () => {
+    expect(versions.length).toBeGreaterThan(1)
+    versions.forEach((version) => {
+      const docsDir = path.join(
+        contentDir,
+        version,
+        originalSourceLocale,
+        'doc'
+      )
+      const files = walk(docsDir, { directories: false })
+      expect(files.length).toBeGreaterThan(0)
+      expect(files.every((file) => file.relativePath.endsWith('.md')))
+    })
   })
 })
 
-xtest('includes translated content for all major versions', () => {
-  // pending Crowdin integration
-})
+describe('translated content', () => {
+  test('saves original structure', () => {
+    versions.forEach((version) => {
+      const languages = fs.readdirSync(path.join(contentDir, version))
+      const originalPath = path.join(
+        contentDir,
+        version,
+        originalSourceLocale,
+        'doc'
+      )
+      const originalFiles = walk(originalPath, { directories: false }).map(
```

Comment:
Not very important, but if you use `walk` instead of `walk.entries`, it will return file paths as strings instead of objects, and you won't need to do this `map`.